workflow/views.py

```diff
@@ -332,6 +332,7 @@ def reportingperiod_update(request, pk):
             pass
         elif (program.last_time_aware_indicator_start_date and
               reporting_period_end.date() < program.last_time_aware_indicator_start_date):
+            print('last start date', program.last_time_aware_indicator_start_date)
```

Comment:
Did you want to keep this print statement or is it left over from your work?

Reply:
Ah, good catch, thanks!